test/unit/alignment/pairwise/fixture/local_affine_banded.hpp

```diff
@@ -52,8 +52,11 @@ static auto dna4_01 = []()
         11,
         "GTTTA",
         "GTCTA",
-        seqan3::alignment_coordinate{column_index_type{5u}, row_index_type{2u}},
-        seqan3::alignment_coordinate{column_index_type{10u}, row_index_type{7u}},
+        // begin and end positions
```

Comment:
You could add this comment in the other tests aswell 
eg `global_affine_unbanded.hpp`